### PR TITLE
 Rename tuple and tuple3 functions to pair and triple. 

### PR DESCRIPTION
**Pull request by harrysarson**
See #48.

I have a slight preference for going with `triplet` over `triple`. This PR uses `triple` although it would be easy for me to change that if folk are keen.

### CODE REVIEW

**Comment by harrysarson** [comment]
When reviewing please double check that I have spelt triple correctly throughout this patch as I had a major battle with dyslexia whilst putting this together and am not entirely confident I won.

File: tests/src/FuzzerTests.elm [hunk 1]
```diff
@@ -17,7 +17,7 @@ die =
 fuzzerTests : Test
 fuzzerTests =
     describe "Fuzzer methods that use Debug.crash don't call it"
-        [ describe "FuzzN (uses tupleN) testing string length properties"
+        [ describe "FuzzN (uses use pair or replacmeForTriplet) testing string length properties"
```
Comment (mgold) [inline tests/src/FuzzerTests.elm]: `(uses pair and triple)`

**Comment by drathier** [comment]
`triple` is what I'd reach for right now, but `triplet` is more correct and my habits will change over time, so I'd be happy with `triplet`. `triple` means three times something, and a `triplet` is three closely related things.

**Comment by harrysarson** [comment]
Googling suggests trio or triple.

https://english.stackexchange.com/a/39825

**Comment by drathier** [comment]
Good point, I now prefer `triple` again. https://english.stackexchange.com/questions/39824/is-triple-the-proper-counterpart-of-pair-when-describing-a-group-of-three-item/39825#comment929406_337695

**Comment by harrysarson** [comment]
Let's go with triple then

**Comment by mgold** [comment]
Sure, I just need the one review comment addressed before merging.

File: tests/src/FuzzerTests.elm (hunk 1 above)
Comment (harrysarson) [reply tests/src/FuzzerTests.elm]: Done!

**Comment by harrysarson** [comment]
@mgold fixed it!

You may want to squash when you merge - if only to fix the spelling of triple in the commit messages!

**Pull request merged**